DMX.Core.Api/Services/Orchestrations/LabOrchestrationService.Exceptions.cs

```diff
@@ -48,6 +51,55 @@ private async ValueTask<List<Lab>> TryCatch(ReturningLabsFunctions returningLabs
             }
         }
 
+        private async ValueTask<Lab> TryCatch(ReturningLabFunctions returningLabFunctions)
+        {
+            try
+            {
+                return await returningLabFunctions();
+            }
+            catch (NullLabException nullLabException)
+            {
+                throw CreateAndLogValidationException(nullLabException);
+            }
+            catch (InvalidLabException invalidLabException)
+            {
+                throw CreateAndLogValidationException(invalidLabException);
+            }
+            catch (ExternalLabDependencyException externalLabDependencyException)
+            {
+                throw CreateAndLogDependencyException(externalLabDependencyException);
+            }
+            catch (ExternalLabServiceException externalLabServiceException)
+            {
+                throw CreateAndLogDependencyException(externalLabServiceException);
+            }

```

Comment:
Why do you need those here? This exception is never emitted from that Add function right?